assets/js/components/labels/UpdateLabelModal.jsx

```diff
@@ -220,6 +227,32 @@ class UpdateLabelModal extends Component {
               <Text strong style={{ fontSize: 16 }}>Note: It only takes a single Label with ADR allowed to apply this setting for devices.</Text>
             </div>
           </TabPane>
+          <TabPane tab="CF List" key="cflist">
+            <div style={{ padding: '30px 50px'}}>
+              <div style={{ marginBottom: 20, display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
+                <Switch
+                  onChange={cfListValue => this.setState({ cfListValue })}
+                  checked={cfListValue}
+                  style={{ marginRight: 8 }}
+                />
+                <Text strong style={{ fontSize: 16 }}>Join-Accept CF List Enabled</Text>
```

Comment:
On my flows one I was including `(applicable to US915 devices only)` since moogey said to make it clear, should we do that here too?

Reply:
good point, adding